README.md

```diff
@@ -3,14 +3,19 @@
 
 # #metrics-webapp
 
-> It's a mobile web application to check a list of metrics (numeric values)
+> It's a mobile web application to display the list of characters for the Bob's Gurger Anime show. Fetcher from the Bob's Burger API and Built using Reactjs, Redux, HTML/CSS, Axios
 
 
 # Sreenshot
+![](https://github.com/Bateyjosue/metrics-webapp/blob/dev/mockup.png)
 
 # Live Demo
 
-> [Go live]()
+> [Go live](https://bob-burger-jb.netlify.app/)
+
+# Live Video
+
+> [See Video](https://www.loom.com/share/706eba1fd58543f5ab30f84e8524f5b0)
 
 # Built With:
 
```

Comment:
Please write a more descriptive and professional acknowledgment that specifies why you are acknowledging Nelson Sakwa.

You could write something like this.

***I would like to thank [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). The design was utilized in accordance with the [Creative Commons License](https://creativecommons.org/licenses/).***